conbench/entities/benchmark_result.py

```diff
@@ -56,7 +56,7 @@ class BenchmarkResult(Base, EntityMixin):
     iqr = Nullable(s.Numeric, check("iqr>=0"))
     error = Nullable(postgresql.JSONB)
     validation = Nullable(postgresql.JSONB)
-    is_step_change = NotNull(s.Boolean, default=False)
+    change_annotations = NotNull(postgresql.JSONB, server_default="{}")
```

Comment:
Is it more useful to make it nullable or have a default of `{}`? Not sure which is simpler to query

Reply:
I'm not sure -- I did some basic googling earlier today and couldn't quickly find an answer. To me it "feels like" the {} is better because my Python code can always use `result.change_annotations.get("asd")` instead of something like `(result.change_annotations or {}).get("asd")`

Reply:
After a lot of discussion with @ElenaHenderson , I made this nullable. It's just simpler to do the migration that way.